src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -465,6 +467,7 @@ def file_issue(testcase,
   issue.reporter = user_email
 
   recovered_exception = None
+
```

Comment:
Can you please limit the scope of the changes and exclude whitespace changes like these?

Reply:
it my fmt, removing